### PR TITLE
tracert: add page

### PR DESCRIPTION
**Pull request by Magrid0**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/windows/tracert.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+> Receive information about each step in the route between your PC and the target.
+> More information: <https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-xp/bb491018(v=technet.10)?redirectedfrom=MSDN>.
+
+- Trace a route:
+
+`tracert {{IP}}`
+
+- Prevent tracert from resolving IP addresses to hostnames:
+
+`tracert -d {{IP}}`
```
Comment (kbdharun) [inline pages/windows/tracert.md:12]:
```suggestion
`tracert /d {{IP}}`
```

File: pages/windows/tracert.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# tracert
+
+> Receive information about each step in the route between your PC and the target.
+> More information: <https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-xp/bb491018(v=technet.10)?redirectedfrom=MSDN>.
+
+- Trace a route:
+
+`tracert {{IP}}`
+
+- Prevent tracert from resolving IP addresses to hostnames:
```
Comment (kbdharun) [inline pages/windows/tracert.md:10]:
```suggestion
- Prevent `tracert` from resolving IP addresses to hostnames:
```

File: pages/windows/tracert.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`tracert -d {{IP}}`
+
+- Force tracert to use IPv4 only:
```
Comment (kbdharun) [inline pages/windows/tracert.md:14]:
```suggestion
- Force `tracert` to use IPv4 only:
```

File: pages/windows/tracert.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`tracert -4 {{IP}}`
```
Comment (kbdharun) [inline pages/windows/tracert.md:16]:
```suggestion
`tracert /4 {{IP}}`
```

File: pages/windows/tracert.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Force tracert to use IPv6 only:
```
Comment (kbdharun) [inline pages/windows/tracert.md:18]:
```suggestion
- Force `tracert` to use IPv6 only:
```

File: pages/windows/tracert.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`tracert -6 {{IP}}`
```
Comment (kbdharun) [inline pages/windows/tracert.md:20]:
```suggestion
`tracert /6 {{IP}}`
```

File: pages/windows/tracert.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# tracert
+
+> Receive information about each step in the route between your PC and the target.
+> More information: <https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-xp/bb491018(v=technet.10)?redirectedfrom=MSDN>.
```
Comment (kbdharun) [inline pages/windows/tracert.md:4]:
```suggestion
> More information: <https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/tracert>.
```

File: pages/windows/tracert.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+`tracert -6 {{IP}}`
+
+- Show detailed help:
+
+`tracert /?`
+
+- Specify the maximum number of hops in the search for the target:
+
+`tracert -h {{max_hops}} {{IP}}`
```
Comment (kbdharun) [inline pages/windows/tracert.md:28]:
```suggestion
- Specify the maximum number of hops in the search for the target:

`tracert /h {{max_hops}} {{IP}}`

- Display help:

`tracert /?`
```

`Display help` is the generic description we use in most of our pages, also we list it at last.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/windows/tracert.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
+# tracert
+
+> Receive information about each step in the route between your PC and the target.
+> More information: <https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/tracert>.
```
Comment (kbdharun) [inline pages/windows/tracert.md:4]:
```suggestion
> More information: <https://learn.microsoft.com/windows-server/administration/windows-commands/tracert>.
```
Just a minor suggestion and the PR is GTG.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**